src/Sarif/ToDotNet/ToDotNet.targets

```diff
@@ -4,11 +4,6 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- Inject object model building. Must be after Microsoft.CSharp.targets is imported. -->
-    <CompileDependsOn>BuildAndInjectObjectModel;$(CompileDependsOn)</CompileDependsOn>
-  </PropertyGroup>
```

Comment:
We no longer build this target in the course of the compilation, so this isn't necessary.